Doppler.Currency/Startup.cs

```diff
@@ -79,21 +75,6 @@ public void ConfigureServices(IServiceCollection services)
                 });
         }
 
-        private void AddServiceSettings(IServiceCollection services)
-        {
-            var dofSettings = new CurrencySettings();
-            Configuration.GetSection("CurrencyCode:DofService").Bind(dofSettings);
-            services.AddSingleton(dofSettings);
-
-            var bnaSettings = new CurrencySettings();
-            Configuration.GetSection("CurrencyCode:BnaService").Bind(bnaSettings);
-            services.AddSingleton(bnaSettings);
```

Comment:
https://github.com/FromDoppler/doppler-currency/pull/16/files#r408873859
> How are configured this services now?
> I do not fine any other reference in this PR.

@elsupergomez 

In this line [link](https://github.com/FromDoppler/doppler-currency/blob/8e4765b3d06b1eec44fbc94122d2c026c1ad3662/Doppler.Currency/Startup.cs#L33) I configure this setting without adding a Singleton